### PR TITLE
:lipstick: Solved ui issue

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206469248179168

### CODE REVIEW

File: views/components/pool-title-bar/index.tsx [hunk 1]
```diff
@@ -19,9 +19,11 @@ const PoolTitleBar: FC<PoolTitleBarProps> = ({
     display="flex"
     maxWidth="65rem"
     alignItems="center"
-    borderRadius="2rem"
     bg="lowestContainer"
-    my={['5xl', '5xl', '5xl', '8xl']}
+    borderRadius="0.5rem"
```
Comment (MarioBatalha) [inline views/components/pool-title-bar/index.tsx:23]: Use value from theme, in this case "xs".

File: views/create-pool/create-pool-form/index.tsx [hunk 2]
```diff
@@ -26,9 +26,9 @@ const CreatePoolForm: FC = () => {
       mx="auto"
       width="100%"
       display="flex"
-      borderRadius="m"
+      borderRadius="0.5rem"
```
Comment (MarioBatalha) [inline views/create-pool/create-pool-form/index.tsx:29]: The same here

File: views/pool-details/pool-form/index.tsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ const PoolForm: FC = () => {
       p={['m', 'm', 'm', '4xl']}
       flexDirection="column"
       bg="lowestContainer"
-      borderRadius="2rem"
+      borderRadius="0.5rem"
```
Comment (MarioBatalha) [inline views/pool-details/pool-form/index.tsx:60]: The same here "xs"

File: views/pool-details/pool-transaction/components/card-wrapper/index.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ const PoolTransactionCardWrapper: FC<PropsWithChildren<CardWrapperProps>> = ({
   children,
 }) => {
   return (
```
Comment (MarioBatalha) [inline views/pool-details/pool-transaction/components/card-wrapper/index.tsx:11]: Remove return

File: views/pool-details/pool-transaction/components/card-wrapper/index.tsx [hunk 5]
```diff
@@ -9,7 +9,7 @@ const PoolTransactionCardWrapper: FC<PropsWithChildren<CardWrapperProps>> = ({
...
-    <Box bg="lowestContainer" borderRadius="2rem" p={['m', 'm', 'm', 'xl']}>
+    <Box bg="lowestContainer" borderRadius="0.5rem" p={['m', 'm', 'm', 'xl']}>
```
Comment (MarioBatalha) [inline views/pool-details/pool-transaction/components/card-wrapper/index.tsx:12]: Use value from theme in border Radius, in this case "xs"

File: views/pools/index.tsx [hunk 6]
```diff
@@ -15,23 +15,20 @@ const Pools: FC = () => {
...
   return (
-    <Layout>
-      <Typography my="2xl" size="large" variant="display" textAlign="center">
-        Pools
-      </Typography>
+    <Layout title="Pools">
       <Box
         p={['s', 's', 's', 'l']}
-        borderRadius="l"
+        borderRadius="0.5rem"
```
Comment (MarioBatalha) [inline views/pools/index.tsx:21]: The same here

File: views/pools/pool-card/index.tsx [hunk 7]
```diff
@@ -17,7 +17,7 @@ const PoolCard: FC<PoolCardProps> = ({ token0, token1, dex, poolObjectId }) => {
       gap="xs"
       display="flex"
       bg="lowestContainer"
-      borderRadius="1rem"
+      borderRadius="0.5rem"
```
Comment (MarioBatalha) [inline views/pools/pool-card/index.tsx:20]: The same here

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by MarioBatalha — APPROVED** [review]

File: views/pool-details/pool-form/index.tsx [hunk 8]
```diff
@@ -57,7 +57,7 @@ const PoolForm: FC = () => {
       p={['m', 'm', 'm', '4xl']}
       flexDirection="column"
       bg="lowestContainer"
-      borderRadius="2rem"
+      borderRadius="xs"
       gap="1.5rem"
```
Comment (git-marcopitra) [inline views/pool-details/pool-form/index.tsx:61]: take care with this `gap` for the next time

File: views/pool-details/pool-transaction/components/card-wrapper/index.tsx [hunk 9]
```diff
@@ -7,20 +7,18 @@ const PoolTransactionCardWrapper: FC<PropsWithChildren<CardWrapperProps>> = ({
...
+  <Box bg="lowestContainer" borderRadius="xs" p={['m', 'm', 'm', 'xl']}>
+    <Box display="flex" justifyContent="space-between" mb="m">
+      <Typography variant="label" size="large">
+        {title}
+      </Typography>
+      {Suffix}
     </Box>
-  );
-};
+    <Box display="flex" flexDirection="column" gap="0.5rem">
```
Comment (git-marcopitra) [inline views/pool-details/pool-transaction/components/card-wrapper/index.tsx:18]: Fix this gap to theme key

File: views/pools/header/action-group.tsx [hunk 10]
```diff
@@ -10,7 +10,7 @@ import { ActionGroupProps } from './header.types';
 const ActionGroup: FC<ActionGroupProps> = ({ showSearchView }) => {
   const { push } = useRouter();
 
-  const gotoFindPool = () => push(Routes[RoutesEnum.FindPool]);
+  const gotoFindPool = () => push(Routes[RoutesEnum.CreatePool]);
```
Comment (git-marcopitra) [inline views/pools/header/action-group.tsx:13]: Do not forget to keep Find Pool when u recover the Find Pool Page

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: views/pools/header/action-group.tsx (hunk 10 above)
Comment (KipandaJr) [reply views/pools/header/action-group.tsx:13]: Solved in other branch/PR

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**